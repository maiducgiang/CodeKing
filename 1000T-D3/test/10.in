4
9338 6057 1720 3700
4044 1901 7066 9609
1560 3584 793 4664
8494 8435 1833 7807
