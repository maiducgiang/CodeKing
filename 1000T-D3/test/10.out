26479400
33409147
4079192
31030857
